.travis.yml

```diff
@@ -20,15 +20,15 @@ branches:
     - master-last_success
     - release-*
 language: node_js
-node_js: "12"
+node_js: "12.18.2"
```

Comment:
We should keep this generic.  Does it use the latest if we have 12 or do we need 12.x?

Reply:
`12 latest 12.x release` 
I'll revert that, it will be generic enough 
12 latest 12.x release